### ISSUE TITLE
Visible focus lost when tab into the (iframe) content

### ISSUE THREAD

**Issue by becka11y** [readium/readium-js-viewer#362]
If you are using the tab key to navigate by keyboard, visible focus is lost when you tab from the last button in the toolbar into the content area iframe. This needs to be made visible to the user knows where focus is currently located.

**Comment by danielweck** [comment]
Hello Becky, I produced a new pull request based on your original code (now merges fine in refactored develop branch):
https://github.com/readium/readium-js-viewer/pull/371/files

**Comment by becka11y** [comment]
I think this should be closed because Daniel merged a fix?

**Comment by danielweck** [comment]
Thanks for the heads-up Becky :)
